onchain/wallet/wallet.go

```diff
@@ -119,10 +121,29 @@ type Wallet struct {
 }
 
 type Config struct {
-	DataDir  string
-	Network  *boltz.Network
-	Debug    bool
-	Electrum onchain.ElectrumConfig
+	DataDir                  string
+	Network                  *boltz.Network
+	Debug                    bool
+	Electrum                 onchain.ElectrumConfig
+	AutoConsolidateThreshold uint64
+	MaxInputs                uint64
+}
+
+func (c *Config) Validate() error {
+	if c.MaxInputs == 0 {
+		c.MaxInputs = MaxInputs
```

Comment:
0 should disable auto consolidation

Reply:
This value isnt configurable and would create errors when sending normally aswell if jt was set to zero

Reply:
Ah, nvmd